docs/development.rst

```diff
@@ -448,6 +448,145 @@ finally force-push to the remote topic branch on your fork::
+Arabidopsis thaliana and a final line of code that registers the species in the catalog.
+
+.. code-block:: python
+
+    _gen_time_citation = stdpopsim.Citation(
+        doi="https://doi.org/10.1890/0012-9658(2002)083[1006:GTINSO]2.0.CO;2",
+        year="2002",
+        author="Donohue",
+        reasons={stdpopsim.CiteReason.GEN_TIME})
+
+    _pop_size_citation = stdpopsim.Citation(
+            doi="https://doi.org/10.1016/j.cell.2016.05.063",
+            year="2016",
+            author="1001GenomesConsortium",
+            reasons={stdpopsim.CiteReason.POP_SIZE})
+
+    _species = stdpopsim.Species(
+        id="AraTha",
+        name="Arabidopsis thaliana",
+        common_name="A. thaliana",
+        genome=_genome,
+        generation_time=1.0,
+        generation_time_citations=[_gen_time_citation],
+        population_size=10**4,
+        population_size_citations=[_pop_size_citation]
+        )
+
+    stdpopsim.register_species(_species)
+
+Once all of this is done, go to the `catalog/__init__.py` file and add a line like the
+one below using the six-letter species identifier. Make sure to keep the comment to
+prevent linting issues.
+
+.. code-block:: python
+
+    from .catalog import PonAbe  #NOQA
+
+--------------------
+Adding a genetic map
+--------------------
+Some species have sub-chromosomal recombination maps available. If there is one for your
+species that you wish to include it is specified in a space delimited file with four
+columns: Chromosome, Position(bp), Rate(cM/Mb), and Map(cM). Each chromosome should be
+placed in a seperate file and with the chromosome id in the file name in such a way that
+it can be programatically parsed out. IMPORTANT: chromosome ids must match those provided
+in the genome definition exactly! Below is an example start to a recombination map file::
+
+    Chromosome Position(bp) Rate(cM/Mb) Map(cM)
+    chr1 32807 5.016134 0
+    chr1 488426 4.579949 0
+
+Once you have the recombination map files formatted, tar and gzip them into a single
+compressed archive. This file will be sent to one of the `stdpopsim` uploaders for
+placement in the AWS cloud once the new genetic map(s) are approved. Finally, you must
+add a `GeneticMap` object to the file named for your species in the `catalog` directory
+(the same one in which the genome is defined) as shown below:
+
+.. code-block:: python
+
+    _genetic_map_citation = stdpopsim.Citation(
+            doi="FILL_ME",
+            author="FILL_ME",
+            year=9999,
+            reasons={stdpopsim.CiteReason.GEN_MAP})
+
+    _gm = stdpopsim.GeneticMap(
+        species=_species,
+        id="FILL_ME", # ID for genetic map, see naming conventions
+        description="FILL_ME",
+        long_description="FILL_ME",
+        url=("https://stdpopsim.s3-us-west-2.amazonaws.com/genetic_maps/dir/filename"),
+        file_pattern="name_{id}_more_name.txt", # a pattern the extracts the chomosome id
```

Comment:
The comment here is a little confusing, I think. (It looks like the GeneticMap docstring for this field is somewhat awkwardly worded too.) How about `recombination map filename, where '{id}' will be replaced with the 'id' field of a given chromosome.`